TypeCobol/Compiler/SqlAnalysis/SqlAnalyzerProvider.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using TypeCobol.Analysis;
+using TypeCobol.Compiler.Directives;
+using TypeCobol.Compiler.Text;
+
+namespace TypeCobol.Compiler.SqlAnalysis
+{
+    /// <summary>
+    /// Class For Sql Analyzer Provider
+    /// </summary>
+    public class SqlAnalyzerProvider : IAnalyzerProvider
```

Comment:
To private analyzer project.

Reply:
idem

Reply:
done in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)